### PR TITLE
D sminorcorrections

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Minor changes

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
Are these images added to the descriptions?

File: DayrSuryan/DSEthiop19.xml [hunk 1]
```diff
@@ -273,14 +272,20 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                               <foreign xml:lang="gez">፥</foreign> is used with the middle dot in red.
-                         </seg>/
+                         </seg>
                            <date>20th century.</date>
                            <desc>Irregular, crude; by a poorly trained scribe.
-                             Most probably the same hand as <ref type="hand" corresp="DSEthiop18#h1">Hand 1 of DS Ethiop. 18</ref>. </desc>
+                             The scribe executed the word divider <foreign xml:lang="gez">፡</foreign>
+                             in such a peculiar way that he interpreted as dot
+                             the last letter element protruding to the right,
+                             (cp., e.g., such letters as ኒ, ዱ, ተ, ኪ, but also ም), and
```
Comment (thea-m) [inline DayrSuryan/DSEthiop19.xml:284]: cp., e.g., I find a bit much :) "such" to me means already that they are a selection, so maybe e.g. can go?

**Review by thea-m — CHANGES_REQUESTED** [review]

**Review by thea-m — COMMENTED** [review]
small formulation question

**Comment by thea-m** [comment]
What images do you mean? In the pdf there is one image of DS Ethiop. 19, which does not show the famous dot

**Comment by thea-m** [comment]
Now I see there was a change of image in the last image file

File: DayrSuryan/DSEthiop19.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop19.xml:284]: ok

**Comment by DenisNosnitsin1970** [comment]
The image of Ethiop 19 does have the dots.

**Comment by DenisNosnitsin1970** [comment]
Yes, there was a change in the last images.

**Comment by DenisNosnitsin1970** [comment]
DO you see the famous dot? :)

**Comment by thea-m** [comment]
Yes, now I see it!

File: DayrSuryan/DSEthiop19.xml (hunk 1 above)
Comment (thea-m) [reply DayrSuryan/DSEthiop19.xml:284]: I'll change it in the file

**Review by thea-m — APPROVED** [review]

**Pull request merged**